website/server/controllers/api-v3/members.js

```diff
@@ -57,6 +57,288 @@ api.getMember = {
   },
 };
 
+function _addAchievement(achievementsResult, type, key, title, text, icon, category, value, earned) {
```

Comment:
Also I get this from eslint 
`
[eslint] 
Expected property shorthand. (babel/object-shorthand)
(property) value: any
`

so that I have to use 
` {
type,
key,
title
}` ?